### DISCUSSION TITLE
December Monthly Challenge Site description.

### DISCUSSION THREAD

**Discussion by BekahHW** [Virtual-Coffee/VC-Community-Docs#227] [Monthly Challenges]
We need to get the new challenge up on the site.  Here's what we need for our `Creative Community Challenge`

## General Description
_2-3 sentence overview_

## Theme: 
_honestly, idk how the theme differs from the title. Be creative? Chill & Create?_

## Challenge Description
- what's the challenge
- what's an acceptable submission
- what constitutes completing the challenge
- are there different roles you can take?
- will there be a VC the musical?

## How to participate
- are there weekly check-ins
- what channel will house the challenge?
- what if someone needs help?
- will we have a talent show/demo/learn how to.... session?

## Resources
- resources that could support members in completing the challenge

**Comment by tkshill** [comment]
## General Description

Devs are more than just the code we write. This month is all about embracing the aspects of yourself beyond the text editor. Share the art, music, poetry, sports, games, or any other hobbies that spark joy for you.  We spend so much time grinding away on understanding things in the tech space. Let's make some space for the other parts of ourselves.

## Challenge description

- We're encouraging folks to spend time this month working on things that aren't necessarily code specific, or using code to improve other hobbies and outlets.
- Let the rest of VC know what you've got going on. You don't have to share your work, but we'd love to see and celebrate your achievements.
- It can also be a great time to explore new hobbies and activities. If someone is doing something your interested in, this is the month to learn more and maybe give it a try.

## How to participate
- Pick one or two non-coding activities you want to focus on and talk about this month.
- Use the VC monthly-challenge channel to post any questions, content or kudos surrounding your hobbies and passions.
- Tag our official account on social media.
- Tweets, blog posts, pictures are all welcome as long as they abide by our code of conduct.
- As always you can reach out to the maintainers or monthly challenge facilitators if you have any questions.
- We'll have opportunities for folks to check in and give updates on how their activities are progressing.

**Comment by abbeyperini** [comment]
General Description
Devs are more than just the code we write. This month is all about embracing the aspects of yourself beyond the text editor. Give back to yourself by indulging in something just for fun. Share the art, music, poetry, sports, games, or any other hobbies that spark joy for you. We spend so much time grinding away on understanding things in the tech space. Let's make some space for the other parts of ourselves.

Challenge description
We're encouraging folks to spend time this month working on things that aren't necessarily code specific, or using code to improve other hobbies and outlets.
Let the rest of VC know what you've got going on. You don't have to share your work, but we'd love to see and celebrate your achievements.
It can also be a great time to explore new hobbies and activities. If someone is doing something your interested in, this is the month to learn more and maybe give it a try.
How to participate
Pick one or two non-coding activities you want to focus on and talk about this month.
Use the VC monthly-challenge channel to post any questions, content or kudos surrounding your hobbies and passions.
Tag our official account on social media.
Tweets, blog posts, pictures are all welcome as long as they abide by our code of conduct.
As always you can reach out to the maintainers or monthly challenge facilitators if you have any questions.
Keep an eye out for the #WIPWednesday thread!
